CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 6.2.0 - Feature `headers` option in `getEntitiesOfTable`
+
+* Added option to `getEntitiesOfTable` to be able to set headers for the http requests.
```

Comment:
the http request

Reply:
It sends more than one http request. So thats why it's plural.